Documentation/CodeSnippets/TypoScriptSyntax/Conditions5.rst.txt

```diff
@@ -0,0 +1,12 @@
+.. Automatic screenshot: Remove this line if you want to manually change this file
+
+.. code-block:: typoscript
+   :caption: Extension examples, file Configuration/TypoScript/Syntax/Conditions5/setup.typoscript
+
+   # Invalid: Conditions must not be used within code blocks
+   someIdentifier {
+      someProperty = foo
+      [frontend.user.isloggedIn]
+         someProperty = bar
+      [GLOBAL]
+   }
```

Comment:
This could be referenced from [CodeSnippets/TypoScriptSyntax/CodeBlockInvalidCondition.rst.txt](https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-CoreApi/pull/2818/files#diff-7ecffe7168307c2890169bf42b6733455b068f2473bc1234f1dd4764978f10f5)

Reply:
This is a code-snippet, we cannot refer one code snipped vfrom another